lib/utils/rruleUtil.js

```diff
@@ -0,0 +1,63 @@
+'use strict'
+
+let packageFound = false;
+try {
+  require('rrule');
+  packageFound = true;
+}
+catch (e) {
+  console.log('no package found');
```

Comment:
If user attempts to use `rruleUtil` and package is not installed, error should not be swallowed, it should be properly thrown. Error message should be much more explicit, it should explain that user needs to install package `rrule` if they want to use this functionality.

Reply:
Yeah that is valid, ill change it